src/sql/base/browser/ui/panel/media/panel.css

```diff
@@ -12,6 +12,16 @@
 	flex-direction: column;
 }
 
+/*High contrast theme for border top*/
+.hc-black .tabbedPanel {
+	border-top-color: #2b56f2;
```

Comment:
You should be using the color registry for this instead of hardcoding the values : 

https://github.com/microsoft/azuredatastudio/blob/main/src/vs/platform/theme/common/colorRegistry.ts#L184